### DISCUSSION TITLE
Dedicated Subdomain in JS.ORG

### DISCUSSION THREAD

**Discussion by warengonzaga** [warengonzaga/buymeacoffee.js#30] [Brainstorm]
I decided to apply for a subdomain to [JS.ORG](https://js.org). It is good to see that we have a dedicated professional open source project landing page in the JS domain name. I'm thinking of using this URL ``buymeacoffee.js.org`` in the future. 🤔

**Motivation**

For me, it is more professional than my original link for it as ``buymeacoffeejs.warengonzaga.com`` which is not good looking for me as the project represents the community contribution even if I'm the one who created it. Also as being the official NodeJS wrapper for BuyMeaCoffee API I think that would be a good reason to get a professional open source subdomain under JS.ORG.

Any comments about this update are appreciated in advance! 👋

**Comment by thomasbnt** [comment]
`buymeacoffee.js.org` is perfect for the subdomain. Maybe an area specific to the project in the future?

**Reply by warengonzaga** [reply]
> `buymeacoffee.js.org` is perfect for the subdomain. Maybe an area specific to the project in the future?

I'm thinking the same way as Discord JS or ``discord.js.org``

**Comment by warengonzaga** [comment]
Added the PR to get a JS.ORG subdomain.
👉 https://github.com/js-org/js.org/pull/6513

This would be a great and professional way to make our official API wrapper stands out.

**Reply by warengonzaga** [reply]
The request has been accepted! 🎉🎉🎉
👉 https://github.com/js-org/js.org/pull/6513#pullrequestreview-784770526

**Reply by thomasbnt** [reply]
Youhouu 🎉

**Comment by warengonzaga** [comment]
I decided to use two domains for this project.

Vercel = ``buymeacoffeejs.warengonzaga.com``
GitHub = ``buymeacoffee.js.org`` - the main one.

**Comment by warengonzaga** [comment]
Congrats to the community! Specially @thomasbnt and @angelofallars for the effort working on the update.
Thanks to the JS.ORG team for giving us the ``buymeacoffee.js.org`` subdomain.  🎉✨🚀

This will allow us to be the trusted and official API wrapper developer for the BMC API.
